src/filefinder.cpp

```diff
@@ -456,6 +456,8 @@ void FileFinder::InitRtpPaths(bool warn_no_rtp_found) {
 #ifdef GEKKO
 	add_rtp_path("sd:/data/rtp/" + version_str + "/");
 	add_rtp_path("usb:/data/rtp/" + version_str + "/");
+#elif defined(SWITCH)
+	add_rtp_path("./rtp/" + version_str + "/");
```

Comment:
could you also add
```
add_rtp_path("/switch/easyrpg-player/" + version_str + "/");
```
?

Just as a fallback in case somebody decides to use a different folder or directly ship a game with it.